TROUBLESHOOTING.md

```diff
@@ -1,15 +1,16 @@
 ### Troubleshooting  
    
-**Powershell 7+ is not installed** (Windows only)
+**Powershell 7+ with AZ module are not installed** (Windows only)
    
 *Symptoms:*  
```

Comment:
@vladborys pls add more generic text in the symptoms section so it also covers the scenario where it is just the AZ module missed, keep the current error message as an example.